backend/db/utils/user_utils.py

```diff
@@ -1,11 +1,27 @@
 import logging
-from sqlmodel import Session, select
+from typing import Optional, Tuple
+from db.user_emails import UserEmails
+from sqlmodel import Session, select, desc
 from db.users import Users 
 from datetime import datetime, timedelta, timezone 
 
 logger = logging.getLogger(__name__)
 
-def user_exists(user) -> bool:
+def get_last_email_date(user_id: str) -> Optional[datetime]:
+    from database import engine
+    """
+    Checks date of user's most recent email 
+
+    """
+    with Session(engine) as session:
+        row = session.exec(
+            select(UserEmails.received_at)
```

Comment:
```suggestion
            
func.max(UserEmails.received_at)
```
This way is cleaner ^ no need to sort